recipes/pyjulia/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.5.7" %}
+
+package:
+  name: pyjulia
+  version: {{ version }}
+
+source:
+  - url: https://files.pythonhosted.org/packages/33/90/ba856cddadd923c94b7ce07dd3623d37cfea22b815076f0c70a47d867a3e/julia-{{ version }}.tar.gz
+    sha256: f24b9788f3e30f33d53fc8854f2fcc54193c84412149cfbc652bde2edd6e8a85
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv && {{ PYTHON }} -c 'import julia; julia.install()'"
+
+requirements:
+  build:
+    - python
+    - pip
+    - julia
+  host:
+    - python >=3.6,<4.0
+    - pip
+    - julia >=1.5.0
+  run:
+    - python >=3.6,<4.0
+    - {{ pin_compatible('julia') }}
+
+test:
+  imports:
+    - julia
+  requires:
+    - pip
+  commands:
+    - pip check
+    - python -c 'import julia; julia.install(); from julia.core import Julia; Julia(compiled_modules=False); from julia import Main'
```

Comment:
Should this actually be `{{ PYTHON }}`?

Reply:
Nah, need not be, I think. But you can change it and see...